### ISSUE TITLE
[BUG]The data in the coord_data_in table is written to user_queries

### ISSUE THREAD

**Issue by Benxiaohai001** [cnosdb/cnosdb#1308]
### Describe the bug

The data in the coord_data_in table is written to user_queries.
version:  1f15c31d4e2f2ecb895ffddb736546d797fc2a67

### To Reproduce

1. start cnosdb 
```bash 
docker run --name dfosssi -d -v /data1/DF_OSS_SI/data/:/data/ xxxxx.aliyuncs.com/cnosdb-yukkit/cnosdb:2.3.1 cnosdb run --config /etc/cnosdb/cnosdb.conf -M singleton
```
2. check user_queries last 10 record.
```sql
select * from user_queries order by time desc limit 10;
```
3. check log
```bash
grep '1204272'  tsdb.log.2023-06-27 
```

### Additional context

```log
usage_schema ❯ select * from user_queries order by time desc limit 10;
+-------------------------------+--------------+--------------+---------+------+---------+
| time                          | database     | host         | node_id | user | value   |
+-------------------------------+--------------+--------------+---------+------+---------+
| 2023-06-27T10:33:26.074395161 | usage_schema | 0.0.0.0:8902 | 1001    | root | 19      |
| 2023-06-27T10:33:26.074393488 | public       | 0.0.0.0:8902 | 1001    | root | 1       |
| 2023-06-27T10:33:26.074133706 | usage_schema | 0.0.0.0:8902 | 1001    | root | 1204272 |
| 2023-06-27T10:33:16.074063313 | usage_schema | 0.0.0.0:8902 | 1001    | root | 19      |
| 2023-06-27T10:33:16.074061524 | public       | 0.0.0.0:8902 | 1001    | root | 1       |
| 2023-06-27T10:33:16.073788387 | usage_schema | 0.0.0.0:8902 | 1001    | root | 1198484 |
| 2023-06-27T10:33:06.074126108 | usage_schema | 0.0.0.0:8902 | 1001    | root | 19      |
| 2023-06-27T10:33:06.074124340 | public       | 0.0.0.0:8902 | 1001    | root | 1       |
| 2023-06-27T10:33:06.073901493 | usage_schema | 0.0.0.0:8902 | 1001    | root | 1192696 |
| 2023-06-27T10:32:56.074815470 | usage_schema | 0.0.0.0:8902 | 1001    | root | 19      |
+-------------------------------+--------------+--------------+---------+------+---------+
Query took 0.173 seconds.

check datanode log 
[root@4e726807b006 log]# grep '1204272'  tsdb.log.2023-06-27 
2023-06-27T10:33:26.074102045Z  INFO coordinator::metrics: lines: [LPLine { measure: "coord_data_in", labels: Labels({"database": "usage_schema", "node_id": "1001", "tenant": "cnosdb"}), value: U64(1204272) }]

```
which is similar to https://stackoverflow.com/questions/76549243/telegraf-encounters-a-data-overwriting-problem-when-writing-data and https://github.com/cnosdb/cnosdb/issues/1300

**Comment by zipper-meng** [comment]
This was fixed in #1301 ,

**Comment by Benxiaohai001** [comment]
version: cnosdb 2.3.1, revision = aa70bf1c570485b6fa0dce5b85546544d9259582
the bug is fixed, so close the issue.
```log
usage_schema ❯ select * from user_queries where database='usage_schema'  order by time desc  limit 10;
+-------------------------------+--------------+--------------+---------+--------+------+-------+
| time                          | database     | host         | node_id | tenant | user | value |
+-------------------------------+--------------+--------------+---------+--------+------+-------+
| 2023-07-10T02:45:40.397854629 | usage_schema | 0.0.0.0:8902 | 1001    | cnosdb | root | 35    |
| 2023-07-10T02:45:30.397748646 | usage_schema | 0.0.0.0:8902 | 1001    | cnosdb | root | 35    |
| 2023-07-10T02:45:20.397146382 | usage_schema | 0.0.0.0:8902 | 1001    | cnosdb | root | 35    |
| 2023-07-10T02:45:10.397652963 | usage_schema | 0.0.0.0:8902 | 1001    | cnosdb | root | 35    |
```
